indexer/bin/indexer.js

```diff
@@ -99,60 +108,81 @@ export default {
       }
 
       return new Response('OK', { status: 200 })
-    } else if (pathname === '/pdp-verifier/pieces-added') {
+    } else if (pathname === '/fwss/piece-added') {
       if (
         !(
-          typeof payload.set_id === 'number' ||
-          typeof payload.set_id === 'string'
+          typeof payload.data_set_id === 'number' ||
+          typeof payload.data_set_id === 'string'
         ) ||
-        !payload.piece_ids ||
-        !Array.isArray(payload.piece_ids) ||
-        !payload.piece_cids ||
-        !Array.isArray(payload.piece_cids)
+        !payload.piece_id ||
+        !(typeof payload.piece_id === 'string') ||
+        !payload.piece_cid ||
+        !(typeof payload.piece_cid === 'string')
```

Comment:
Should I make these validations stricter now? `data_set_id` and `piece_id` and `piece_cid` will be always a string.

Reply:
yes please

Reply:
Done in [5530229](https://github.com/filbeam/worker/pull/301/commits/5530229673c28f07de0600d5dc5899a37ab6aec2)